project_resource_calendar/tests/common.py

```diff
@@ -19,16 +24,25 @@ def setUp(self):
             'resource_type': 'room',
             'allow_double_book': True,
         })
-
         self.instrument_1 = self.Instruments.create({
             'name': 'Test Instrument 1',
             'resource_type': 'material',
             'room_id': self.room_1.id,
             'allow_double_book': True,
         })
-    
         self.weekly_report_wizard_common = self.WeeklyReportWizards.create({
                 'room_id': self.room_1.id,
                 'recurrency': True,
                 'state': 'open',
             })
+        self.room_2 = self.Rooms.create({
+            'name': room_name,
```

Comment:
You can directly put the name without using this variable

Reply:
@rimbendhaou Done